CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ You can find and compare releases at the [GitHub release page](https://github.co
 - Move complexity related code from `FieldDefinition` to `QueryComplexity`
 - Exclude unused standard types from the schema
 - Require lazy type loader to return `Type` directly without an intermediary callable
+- Rename `ServerConfig` option `persistentQueryLoader` to `persistedQueryLoader`
```

Comment:
Must admit I don't know where this is coming from and maybe is not really related to phpstan?

Is it this https://www.apollographql.com/docs/apollo-server/performance/apq/?

Reply:
Not related to PHPStan at all, just felt like making a breaking change 😉  The previously chosen name is inconsistent with the usual terminology, and I think this library is better off in the long run with having the "correct" name in place.